### PR TITLE
feat: Update Copy Button UI

### PR DESCRIPTION
**Pull request by leithonenglish**
Minor update to the `CopyButton` component. Added an updated `Tone` and the ability to pass the size attribute (same size values as the `Button` component)

Before:
Screenshot 2022-05-10 at 6 51 48 PM

After:  
Screenshot 2022-05-10 at 6 51 32 PM

### CODE REVIEW

**Review by pvorozhe — COMMENTED** [review]
Would you mind providing screenshots? LGTM otherwise!

File: src/components/CopyButton/CopyButton.tsx [hunk 1]
```diff
@@ -7,14 +7,7 @@ import { ButtonProps, Button } from "../Button/Button"
...
 const baseCss: ThemeCss = _theme => ({
-  "html:not([dir=rtl]) &": {
-    borderTopLeftRadius: 0,
-    borderBottomLeftRadius: 0,
-  },
-  "html[dir=rtl] &": {
-    borderTopRightRadius: 0,
-    borderBottomRightRadius: 0,
-  },
+  fontWeight: 600,
```
Comment (fk) [inline src/components/CopyButton/CopyButton.tsx:10]:
```suggestion
  fontWeight: _theme.fontWeights.semibold,
```

(probably wanna drop the leading underscore)

File: src/components/CopyButton/CopyButton.tsx [hunk 2]
```diff
@@ -7,14 +7,7 @@ import { ButtonProps, Button } from "../Button/Button"
 import { ThemeCss } from "../../theme"
 
 const baseCss: ThemeCss = _theme => ({
```
Comment (fk) [inline src/components/CopyButton/CopyButton.tsx:9]:
```suggestion
const baseCss: ThemeCss = theme => ({
```

**Review by pvorozhe — APPROVED** [review]

File: src/components/CopyButton/CopyButton.tsx [hunk 3]
```diff
@@ -6,15 +6,8 @@ import copyToClipboard from "../../utils/helpers/copyToClipboard"
...
-  "html:not([dir=rtl]) &": {
-    borderTopLeftRadius: 0,
-    borderBottomLeftRadius: 0,
-  },
-  "html[dir=rtl] &": {
-    borderTopRightRadius: 0,
-    borderBottomRightRadius: 0,
-  },
+const baseCss: ThemeCss = theme => ({
+  fontWeight: _theme.fontWeights.semiBold,
```
Comment (fk) [inline src/components/CopyButton/CopyButton.tsx:10]:
```suggestion
  fontWeight: theme.fontWeights.semiBold,
```

`_theme` is undefined:

image

**Review by fk — APPROVED** [review]
LGTM!

**Pull request merged**